### PR TITLE
Secure URLs (3 of 3)

### PR DESCRIPTION
**Pull request by vszakats**
Ref: https://github.com/Homebrew/homebrew-cask/pull/52975

### CODE REVIEW

File: Casks/quickpvr.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   url "https://www.limbic.com/downloads/QuickPVR_v#{version.major}.zip"
   name 'QuickPVR'
-  homepage 'http://www.limbic.com/quickpvr.html'
+  homepage 'https://www.limbic.com/#games'
```
Comment (commitay) [inline Casks/quickpvr.rb]: This doesn't seem to be the homepage?

File: Casks/soube.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
   url "https://github.com/DracotMolver/Soube/releases/download/#{version}/soube-#{version}.dmg"
   appcast 'https://github.com/DracotMolver/Soube/releases.atom'
   name 'Soube'
-  homepage 'http://soube.diegomolina.cl/'
+  homepage 'https://web.archive.org/web/20171224130516/soube.diegomolina.cl/'
```
Comment (commitay) [inline Casks/soube.rb]: I'd change this to the GitHub repo.

File: Casks/td-agent.rb [hunk 3]
```diff
@@ -2,8 +2,8 @@
   version '2.3.5-1'
   sha256 '2f2973ed29a3edd416cc5dad6b924751a2cdb8cdbb2792dc5344f3e13529b190'
 
-  # packages.treasuredata.com.s3.amazonaws.com was verified as official when first introduced to the cask
-  url "http://packages.treasuredata.com.s3.amazonaws.com/2/macosx/td-agent-#{version}.dmg"
+  # s3.amazonaws.com/packages.treasuredata.com was verified as official when first introduced to the cask
+  url "https://s3.amazonaws.com/packages.treasuredata.com/2/macosx/td-agent-#{version}.dmg"
```
Comment (commitay) [inline Casks/td-agent.rb]: How can we verify this URL from the homepage or appcast?

File: Casks/quickpvr.rb (hunk 1 above)
Comment (vszakats) [reply Casks/quickpvr.rb]: The project is no longer presented on this homepage.

File: Casks/td-agent.rb (hunk 3 above)
Comment (vszakats) [reply Casks/td-agent.rb]: It's a synonym. No need to.

**Comment by vszakats** [comment]
This more and more looks like an impossible task to achieve :( I've been dealing with these for 16 hours straight just today and the situation is worse than ever. At this point I'd prefer to abandon it and push my efforts into the trash.

To keep sanity I'd suggest to pull these as-is and commit follow up fixes (e.g. Cask deletions) where it makes sense, but I strongly think that using an archive.org homepage (or the vendor's generic product page) is better than a broken URL, at least until the Cask is there.

For S3 URLs, my URL use is positively safe, and I don't see them being thrown away on the grounds that the app maintainers didn't configure it with HTTPS in mind.

File: Casks/soube.rb (hunk 2 above)
Comment (vszakats) [reply Casks/soube.rb]: The archive URL has different and more rich content than the GitHub page. In fact it's still being linked from the top of the GitHub page.

**Comment by commitay** [comment]
> This more and more looks like an impossible task to achieve

Hence why dealing them in smaller batches is easier.

> For S3 URLs, my URL use is positively safe, and I don't see them being thrown away on the grounds that the app maintainers didn't configure it with HTTPS in mind.

We have a specific rule to verify URLs, it doesn't that these URLs are safe to use. We can adjust our rules to deal with it but I'm not happy to just merge these as-is.

**Comment by commitay** [comment]
If you want to abandon them I'll deal with them from here.

**Comment by vszakats** [comment]
Please do, thank you — I can't deal with these anymore at this point.

**Comment by commitay** [comment]
Okay, I won't bother preserving the commit attribution when making changes but as the PRs are yours the changes will still be credited to you when we squash and merge.

**Review by commitay — APPROVED** [review]

**Pull request merged**